pkg/kubelet/api/v1alpha1/lifecycle/api.pb.go

```diff
@@ -1812,7 +1812,7 @@ var (
 
 var fileDescriptorApi = []byte{
 	// 568 bytes of a gzipped FileDescriptorProto
-	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x09, 0x6e, 0x88, 0x02, 0xff, 0x9c, 0x54, 0xcd, 0x6e, 0x9b, 0x4c,
+	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0xff, 0x9c, 0x54, 0xcd, 0x6e, 0x9b, 0x4c,
```

Comment:
I don't think you modified the proto, so probably drop this generated file from the patch set.